conbench/tests/api/_asserts.py

```diff
@@ -60,7 +60,8 @@ def assert_400_bad_request(self, r, message):
     def assert_401_unauthorized(self, r):
         assert r.status_code == 401, r.status_code
         assert r.content_type == "application/json", r.content_type
-        assert r.json == {"code": 401, "name": "Unauthorized"}, r.json
+        assert r.json["code"] == 401, r.json
+        assert r.json["name"] == "Unauthorized"
```

Comment:
Note: now the dict also has a `description` key and it's a long  sentence with a description of what this error means; default Flask behavior. Didn't want to put this sentence into the test suite.